aquarium/aquarium.js

```diff
@@ -1535,7 +1543,7 @@ function initialize() {
       var oldDepthTestCap = gl.isEnabled(gl.DEPTH_TEST);
       var oldBlendCap = gl.isEnabled(gl.BLEND);
       var oldScissorTestCap = gl.isEnabled(gl.SCISSOR_TEST);
-      var oldViewport = gl.getParameter(gl.VIEWPORT);
+      var oldViewport = viewport;
       var oldFramebufferBinding = gl.getParameter(gl.FRAMEBUFFER_BINDING);
```

Comment:
Why we still need this?

Reply:
Depending on if VR is enabled, `oldFramebufferBinding` can be `null` or an FBO. It's not handled in the same way as the viewport because the signature of `render()` becomes more ugly otherwise. Fortunately, the framebuffer binding is cached in the renderer process (I believe any `Object` object must be cached), so `getParameter()` is not a big problem here.

Reply:
Ack